docs/machine-learning/how-to-guides/consuming-model-ml-net.md

```diff
@@ -1,32 +1,125 @@
 ---
-title: Operationalize a trained machine learning model in apps - ML.NET
+title: "How-To: Save and Load Data Preparation Pipelines and Trained Models"
 description: Discover how to use ML.NET to consume a trained and evaluated machine learning model in applications
-ms.date: 03/05/2019
+ms.date: 04/23/2019
+author: luisquintanilla
+ms.author: luquinta
 ms.custom: mvc,how-to
 #Customer intent: As a developer, I want to use ML.NET to consume my trained and evaluated machine learning model in my applications.
 ---
-# Operationalize a trained machine learning model in apps - ML.NET
 
-> [!NOTE]
-> This topic refers to ML.NET, which is currently in Preview, and material may be subject to change. For more information, visit [the ML.NET introduction](https://www.microsoft.com/net/learn/apps/machine-learning-and-ai/ml-dotnet).
+# How-To: Save and Load Data Preparation Pipelines and Trained Models
 
-This how-to and related sample are currently using **ML.NET version 0.10**. For more information, see the release notes at the [dotnet/machinelearning github repo](https://github.com/dotnet/machinelearning/tree/master/docs/release-notes)
+Learn how to save and load data preparation pipelines and trained models in your application. Although this sample uses a regression model, the same process applies to other ML.NET algorithms.
 
-When the model metrics look good to you, it's time to 'operationalize' the model. The `model` object you built can be consumed, persisted, and reused in different environments, applying the same steps that it 'learned' during training.
-
-To save the model to a file, and reload it (potentially in a different context), use the following example:
+Throughout the model building process, a model lives in memory and is accessible throughout the application's lifecycle. However, once the application stops running, if the model is not saved somewhere locally or remotely, it is no longer accessible. Typically models are used at some point after training either for inference or re-training. Therefore, it is important to store the model as well as its data preparation pipeline. Using a data preparation and model training pipeline like the one detailed below, save and load these components using the steps described in subsequent sections of this document.
```

Comment:
I wonder if it would be better to use model generically for the most of the doc, and then at the bottom, make the distinction the model and the data preparation pipeline.

So .. structure would be:

Title: Save and Load Models

h2: Save mode locally
h2: Load model from local storage
h2: Load model from remote storage
h2: Save and load data preparation pipeline separately

Reply:
#resolved